src/resources/views/form_content.blade.php

```diff
@@ -73,6 +73,87 @@
                 $('html, body').animate({scrollTop: (fieldOffset - 30)});
             }
           @endif
+
+          @php
+               $unique_fields = array_filter($fields, function($f){
+                   return isset($f['unique']) && $f['unique'];
+               });
+           @endphp
+
+           @if ( count($unique_fields) )
+           //Unique field checks
```

Comment:
Is this comment redundant (it is sitting right underneath a count for something called `$unique_fields`.